### ISSUE TITLE
New ROI UNiTer Training With New inputs

### ISSUE THREAD

**Issue by calcuttj** [WireCell/wire-cell-toolkit#522]

**Comment by calcuttj** [comment]
Now running the UNet-only portion of the training with the outputs created in #521 

condor jobs 
1996.0   jcalcutt        9/2  11:15   0+00:00:08 R  0    0.0 run_training.sh
1997.0   jcalcutt        9/2  11:16   0+00:00:07 R  0    0.0 run_training.sh
1998.0   jcalcutt        9/2  11:16   0+00:00:07 R  0    0.0 run_training.sh


Note: U/V are running on 3 GPUs and W on 2 GPUs all w/ batch size 8. Takes roughly 12 min / epoch for U/V and 13 min/epoch for W

Running all for 25 epochs so 300 (325) min for U/V (W) ~ 5.5 hours. Metrics plots afterwards will take a little bit longer too. Expect this to be done tonight. Can then launch ROI Uniter

**Comment by calcuttj** [comment]
Trainings finished. Started a "legacy" (no restriction on attention) training
2017.0   jcalcutt        9/2  18:05   0+00:00:13 R  0    0.0 run_training.sh


6 gpus, batch 3 / gpu

**Comment by calcuttj** [comment]
I had trained the model after 2017 for 5 more epochs (total 10 epochs) and then unfroze the UNets. Currently running now, looks like eval loss is diverging (maybe overfitting the 16k inputs with 3 UNets + Transformer parameters)

**Comment by calcuttj** [comment]
Running with chunk=8, 4, 2 (how many channels get merged together) and band=0 (how many time ticks are used simultaneously in attention)

2216.0   jcalcutt        9/4  17:53   0+00:00:30 R  0    0.0 run_training.sh
2217.0   jcalcutt        9/4  17:53   0+00:00:00 I  0    0.0 run_training.sh
2218.0   jcalcutt        9/4  18:02   0+00:00:00 I  0    0.0 run_training.sh

**Comment by brettviren** [comment]
Related #529 .  I think here we only have PDHD in mind.  PDVD also needs a model eventually. I think it is okay to let that wait until after the collab meeting unless we happen to get PDHD nailed early.

**Comment by brettviren** [comment]
Info from slack: Jake finds that FFT-downsampling by 4 of the splat leads to ringing and this ringing is learned by roiuniter.  This is due to the bandwidth being greater than the nyquist frequency plus TorchLMN giving power to the nyquist bin.  Using rebin-4 avoids the ringing.  This requires yet another retraining.

At the cost of reworking the Jsonnet, we could generate the splat directly in the 4x downsample domain and remove the need for rebin/resample.  Something for the longer term.
